app/components/shared/cluster/consts.ts

```diff
@@ -16,12 +16,13 @@ export const clusterServiceEnum = Object.freeze({
 });
 export const clusterServices = _.keys(clusterServiceEnum);
 
-export const clusterServiceStatusEnum = Object.freeze({
-    OK: 'OK',
-    Fail: 'Fail',
-    Degraded: 'Degraded',
-    Unknown: 'Unknown'
-});
+export enum clusterServiceStatusEnum {
```

Comment:
I think this would be more in line with existing enum declarations:
```suggestion
export enum ClusterServiceStatus {
```

Reply:
I renamed it: [38c47fb](https://github.com/cloudify-cosmo/cloudify-stage/pull/2223/commits/38c47fb20a64eceddaf095b84b2fd4491d5a5605)